orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/stage_data.py

```diff
@@ -60,9 +64,9 @@ def pre_process_metadata(context: AbstractComputeExecutionContext) -> Nothing:
         "staging_table_expiration_ms": Int
     },
     input_defs=[InputDefinition("start", Nothing)],
-    output_defs=[OutputDefinition(name="staging_dataset_name", dagster_type=String)]
+    output_defs=[OutputDefinition(name="staging_dataset_name", dagster_type=HcaStagingDatasetName)]
```

Comment:
output definition can be omitted for solids that return a single output - its type is inferred from the annotated return type

Reply:
👍 Interestingly, the output gets named "result" when pulling it via output_value() in tests.